src/main/java/blackjack/domain/result/DealerResult.java

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.result;
+
+import blackjack.domain.common.Name;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public class DealerResult {
+    private final Name name;
+    private final EnumMap<ResultStatus, Count> resultStatusBoard;
+
+    public DealerResult(final Name name, final EnumMap<ResultStatus, Count> resultStatusBoard) {
+        this.name = name;
+        this.resultStatusBoard = resultStatusBoard;
+    }
+
+    public static DealerResult of(final Name name, final List<GamePlayerResult> gamePlayerResults) {
+        final EnumMap<ResultStatus, Count> resultStatusBoard = new EnumMap(ResultStatus.class);
```

Comment:
raw type이 있습니다! 그리고 Map interface를 사용하도록 구현해도 될거같아요.

```suggestion
        final Map<ResultStatus, Count> resultStatusBoard = new EnumMap<>(ResultStatus.class);
```